src/main/resources/application.conf

```diff
@@ -0,0 +1,14 @@
+// Token environment to use (DEV, STG, SANDBOX, PRD)
+environment = "SANDBOX"
+
+// Developer key to access API
+devKey = "4qY7lqQw8NOl9gng0ZHgT4xdiDqxqoGVutuZwrUYQsI"
+
+// Desired email for the account. Uses noverify in order to avoid email verification
+email = "merchantA+noverify@example.com"
+
+// Directory where keys will be stored
+keysDir = "./keys"
+
+// Destination IBAN
+destinationIban = "IBAN123456789"
```

Comment:
Needs to be IBAN and account number, right?

Reply:
No, IBAN is the only field in the sepa BankAccount object